index.html

```diff
@@ -1,16 +1,269 @@
+      <ul>
+        <li>Linux</li>
+        <li>Windows</li>
+        <li>macOS</li>
+        <li>FreeBSD</li>
+        <li>Android (planned)</li>
+        <li>QNX (planned)</li>
+        <li>FreeRTOS (planned)</li>
+        <li>VxWorks (planned)</li>
+      </ul>
+    </li>
+    <li>Wide range of language bindings
+      <ul>
+        <li>C and C++ (planned)</li>
+        <li>Python (planned)</li>
+        <li>C# (planned)</li>
+        <li>and more</li>
+      </ul>
+    </li>
+    <li>Host2Host communication (planned)</li>
+    <li>Gateways to network protocols
+      <ul>
+        <li>DDS (planned)</li>
+        <li>gRPC (planned)</li>
+        <li>Zenoh (planned)</li>
+      </ul>
+    </li>
+    <li>Modular design allows custom extension, e.g. GPUs and Hypervisors</li>
+  </ul>
+</td>
+</tr>
+<tr>
+<td><a class="md-button" href="latest">Continue with iceoryx</a></td>
+<td><a class="md-button" href="https://github.com/eclipse-iceoryx/iceoryx2">Continue with iceoryx2</a></td>
+</tr>
+</tbody>
+</table>
+</p>
+
+
+<table>
+<thead>
+<tr>
+<th><h2><span class="twemoji"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 13.5 2.25 12H7.5l-.6-1.5H2L1.25 9h7.8l-.6-1.5H1.11L.25 6H4a2 2 0 0 1 2-2h12v4h3l3 4v5h-2a3 3 0 0 1-3 3 3 3 0 0 1-3-3h-4a3 3 0 0 1-3 3 3 3 0 0 1-3-3H4v-3.5H3m16 5a1.5 1.5 0 0 0 1.5-1.5 1.5 1.5 0 0 0-1.5-1.5 1.5 1.5 0 0 0-1.5 1.5 1.5 1.5 0 0 0 1.5 1.5m1.5-9H18V12h4.46L20.5 9.5M9 18.5a1.5 1.5 0 0 0 1.5-1.5A1.5 1.5 0 0 0 9 15.5 1.5 1.5 0 0 0 7.5 17 1.5 1.5 0 0 0 9 18.5z"/></svg></span> Fast</h2></th>
+<th><h2><span class="twemoji"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.5 11H19V7a2 2 0 0 0-2-2h-4V3.5A2.5 2.5 0 0 0 10.5 1 2.5 2.5 0 0 0 8 3.5V5H4a2 2 0 0 0-2 2v3.8h1.5c1.5 0 2.7 1.2 2.7 2.7 0 1.5-1.2 2.7-2.7 2.7H2V20a2 2 0 0 0 2 2h3.8v-1.5c0-1.5 1.2-2.7 2.7-2.7 1.5 0 2.7 1.2 2.7 2.7V22H17a2 2 0 0 0 2-2v-4h1.5a2.5 2.5 0 0 0 2.5-2.5 2.5 2.5 0 0 0-2.5-2.5z"/></svg></span> Flexible</h2></th>
+<th><h2><span class="twemoji"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21 11c0 5.55-3.84 10.74-9 12-5.16-1.26-9-6.45-9-12V5l9-4 9 4v6m-9 10c3.75-1 7-5.46 7-9.78V6.3l-7-3.12V21z"/></svg></span> Dependable</h2></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><ul><li>True zero-copy shared memory communication</li><li>Message transfers with a latency of less than 1 µs</li><li>Made to handle GBytes/sec data transfers</li></ul></td>
+<td><ul><li>Support of various operating systems, communication patterns and APIs</li><li>Service discovery enables the design of dynamic systems</li><li>Easy to integrate into frameworks like ROS 2 or AUTOSAR Adaptive</li></ul></td>
+<td><ul><li>Developed according to automotive requirements</li><li>Implementation based on static memory and lock-free algorithms</li><li>Huge C++ library with safe STL implementations</li></ul></td>
+</tr>
+</tbody>
+</table>
+
+
+<table align="center">
+<thead>
+<tr>
+<th colspan="2"><h2>Commercial support</h2></th>
```

Comment:
I thought about adding a `*` with a link to https://www.eclipse.org/projects/handbook/#resources-website-services like "Commercial support [*](https://www.eclipse.org/projects/handbook/#resources-website-services)" but I'm not sure about it.

Reply:
@elBoberido I was also unsure reading this bit:

![image](https://github.com/eclipse-iceoryx/iceoryx-web/assets/8661268/87205d72-15dd-4ca5-b0e6-897fcead658a)

However, I think the change is ok as we have clearly separated the two websites, see below:

![image](https://github.com/eclipse-iceoryx/iceoryx-web/assets/8661268/aeb0e73a-3221-4bc8-ab3b-b8770dc6ce76)